eva-lib/src/main/java/uk/ac/ebi/eva/lib/filter/Helpers.java

```diff
@@ -8,14 +10,26 @@
                                                                                         String polyphenScore,
                                                                                         String siftScore,
                                                                                         List<String> studies,
-                                                                                        List<String> consequenceType) {
+                                                                                        List<String> consequenceType,
+                                                                                        List<String> files,
+                                                                                        List<Variant.VariantType> types,
+                                                                                        List<String> alternates) {
```

Comment:
At the moment there are 2 very well delimited filter use cases:

* EVA endpoints: First 5 arguments
* GA4GH endpoints: "files" argument only

So it is better to create 2 methods, one for each case, than a completely generic one that is called with many null arguments.

Another option would be to use a builder object that would be totally flexible:

```java
List<VariantEntityRepositoryFilter> filters = withMaf(maf)
                                             .withPolyphenScore(polyphenScore)
                                             .withSiftScore(siftScore)
                                             .withStudies(studies);
```

This would be most convenient for testing as a lot of combinations are necessary, but not so much for the application code, so the suggestion about the 2 facade methods could still apply.